examples/dt/bmo01/dataplane/nodesets/values.yaml

```diff
@@ -0,0 +1,235 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  network_config_template: &network_config_template |
+    ---
+    {% set mtu_list = [ctlplane_mtu] %}
+    {% for network in nodeset_networks %}
+    {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+    {%- endfor %}
+    {% set min_viable_mtu = mtu_list | max %}
+    network_config:
+      - type: ovs_bridge
+        name: {{ neutron_physical_bridge_name }}
+        mtu: {{ min_viable_mtu }}
+        use_dhcp: false
+        dns_servers: {{ ctlplane_dns_nameservers }}
+        domain: {{ dns_search_domains }}
+        addresses:
+          - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+        routes: {{ ctlplane_host_routes }}
+        members:
+          - type: interface
+            name: enp5s0
```

Comment:
Replacing this interface name is almost impossible, that's why we use nic mappings in CI. Replacing the mapping the Nodeset vars is easy.

Reply:
I switched it to use `nic1` here.